src/scripts/main.js

```diff
@@ -1,3 +1,256 @@
 'use strict';
 
-// write code here
+// 1. Implement table sorting by clicking on the title (in two directions).
+const table = document.querySelector('table');
+let sortDirection = true;
+
+function getTableData() {
+  return [...table.querySelectorAll('tbody tr')].map((row) => {
+    return [...row.children].map((cell) => cell.innerText);
+  });
```

Comment:
Code quality issue: It's a good practice to separate the concerns. Here you are selecting the elements and mapping them in the same function. Consider creating a separate function to select the table rows and another one to map the table data.